keyvi/src/cpp/dictionary/dictionary_merger.h

```diff
@@ -100,9 +100,29 @@ final {
     };
 
 public:
-    DictionaryMerger(size_t memory_limit = 1073741824, const merger_param_t& params = merger_param_t())
```

Comment:
@narekgharibyan Agreed, that makes sense.

Will not change it here, but for the new way of specifying the limit. I suggest to change the key accordingly into 'memory_limit_mb'